plugins/keycloak-backend/README.md

```diff
@@ -76,55 +78,68 @@ When using client credentials, the access type must be set to `confidential` and
 
       and and then use the configured scheduler
 
-      ```diff
+      ```ts
       // packages/backend/src/plugins/catalog.ts
-      + import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';
-
-        export default async function createPlugin(
-          env: PluginEnvironment,
-        ): Promise<Router> {
-          const builder = await CatalogBuilder.create(env);
-
-      +   builder.addEntityProvider(
-      +     KeycloakOrgEntityProvider.fromConfig(env.config, {
-      +       id: 'development',
-      +       logger: env.logger,
-      +       scheduler: env.scheduler,
-      +     }),
-      +   )
+      import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';
+
+      export default async function createPlugin(
+        env: PluginEnvironment,
+      ): Promise<Router> {
+        const builder = await CatalogBuilder.create(env);
+
+        builder.addEntityProvider(
+          KeycloakOrgEntityProvider.fromConfig(env.config, {
+            id: 'development',
+            logger: env.logger,
+            scheduler: env.scheduler,
+          }),
+        )
         ...
-        }
+      }
       ```
 
    2. Add a schedule directly inside the `packages/backend/src/plugins/catalog.ts` file:
 
-      ```diff
+      ```ts
```

Comment:
How is this helpful change? User needs to know which lines to add. Not all are added to enable this plugin. Please respect consistent formatting across our and upstream plugins.

Reply:
The codeblock was a `ts` codeblock before the most recent keycloak [pr](https://github.com/janus-idp/backstage-plugins/commit/dcf345f8f0ca27f85cb4bf158a1d223cb706bb60#diff-74b70d63dd0c250dcfdd7cacf4639fc82a1a66098d9fc2a269bb98bf89790f9fL59). I can change this back to a `diff` codeblock.

Reply:
Yes, it was. However, the block contained `+ ` start of lines, it was a `diff` from the beginning. Please change to `diff`.